dist/pathfora.js

```diff
@@ -1568,7 +1604,7 @@
      * @throws {Error} error
      * @param  {array} array list of widgets to initialize
      */
-    initializeWidgetArray: function (array) {
+    initializeWidgetArray: function (array, accountId) {
```

Comment:
if we are going to force full account id after allowing aid we should do some validation here to surface a warning. should be able to just look for length and assume anything `<=` 4 is an aid that could have installed pathfora manually before the update.